### ISSUE TITLE
Credit where credit is due

### ISSUE THREAD

**Issue by joshgoebel** [exercism/wren#4]
It seems easiest to start by copying many of the great Javascript exercises and then porting them to Wren.  In many cases the instructions may even work as-is (though perhaps this will change as the exercises get harder).  Hopefully many of the test suits will work (after reworking syntax/porting slightly, etc).  The Wren solutions I'm writing from scratch.

So... if I'm starting with the foundation of using JS exercises... what information should go in the `author` and `contributor` blocks for `config.js` for each exercise?  Should I be the new author? Should the contributors from JS be left in place?  Should it all reset to zero because it's now Wren?

I don't want to borrow without giving credit where it's due so I want to do the right thing here whatever that is.

**Comment by joshgoebel** [comment]
@iHiD

**Comment by ErikSchierboom** [comment]
> So... if I'm starting with the foundation of using JS exercises... what information should go in the author and contributor blocks for config.js for each exercise? Should I be the new author? Should the contributors from JS be left in place? Should it all reset to zero because it's now Wren?

You should be the sole author and the JS contributors should be removed, so yes: it should reset. 

To still attribute the original source, we support a `forked_from` property (see [the docs](https://github.com/exercism/docs/blob/main/building/tracks/concept-exercises.md#file-metaconfigjson)) for Concept Exercises, but we didn't yet define it for Practice Exercises. It might make sense to do so judging from your comment. Do you agree @iHiD?

**Comment by iHiD** [comment]
I think because the idea is that PS is the central space, we don't have forked_from.

Are these great JS exercises, or great PS exercises?

**Comment by ErikSchierboom** [comment]
Yes that is indeed the idea, but for the implementer, there is hardly any difference between forking a Concept Eexercise versus a Practice Exercise. In both cases someone else will have done a lot of the groundwork to implement the exercise. So there is an argument to be made that Practice Exercises can also be forked from another track, to reward the forked exercise's author.

**Comment by iHiD** [comment]
That makes sense. But I imagine if you compare say the JS exercises to any other track, they'll be relatively similar? ie, the credit isn't simply due for one existing track that the new track is copying from, but for the whole PS team?

**Comment by ErikSchierboom** [comment]
🤷 They will indeed be similar across tracks, although the implementation is of course still track-specific. But yeah, it is not as straightforward as for Concept Exercises.

**Comment by joshgoebel** [comment]
Any more thoughts here?

**Comment by ErikSchierboom** [comment]
At least for now, just put yourself as the author. We'll look at this in more detail once everything has stabilized.
